detect_variants/vep.cwl

```diff
@@ -25,6 +25,8 @@ inputs:
         inputBinding:
             prefix: "-i"
             position: 1
+    reference:
+        type: File
```

Comment:
I made the reference optional. However, with that change if someone sets `hgvs` to `true` and doesn't specify a reference, the workflow will still dispatch but VEP will fail. 

I tried to implement Tom's suggestion of dependent_parameters but it seems like you can't make the dependent parameter set optional. I tried a few things with Tom's help but couldn't get it to work.